tests/fast_api/test_fast_route.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+
+import aiohttp
+
+
+def test_read_main(client):
+    async def test_async():
```

Comment:
I think the outer function needs to be async and you don't need this inner function at all, just use the async stuff directly in the function